package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cql-exec-fhir",
-  "version": "1.3.1",
+  "version": "1.3.2",
```

Comment:
Since this adds support for a new model (FHIR 4.0.1), I'd consider that a _minor_ release rather than a _patch_ release; it now supports entire CQL libraries it did not support before.  So I'd suggest we bump to 1.4.0.  Would you agree?